docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -41,21 +36,21 @@ There are several aspects to this syntax, described under the following topics i
 
 * **data_type**
 
-    data_type represents the data type of each property. For more information about data types that Nebula Graph supports, see data types section.
-    
+    data_type represents the data type of each property. For more information about data types that Nebula Graph supports, see [data-type](../../1.data-types/data-types.md) section.
+
     > NULL and NOT NULL constrain are not supported yet when creating tags/edges (comparing with relational databases).
 
-* **default values**
+<!-- * **default values**
```

Comment:
why comment out?

Reply:
Code is not ready, the function is not supported yet.